### ISSUE TITLE
Pump water from water source

### ISSUE THREAD

**Issue by ShawnTheShadow** [stijnwop/hoseSystem#61]
Would it be possible to add support where we can run a hose to a water source like a lake/river and the other end to the trailer?

**Comment by stijnwop** [comment]
Yeah that's planned and partly implemented already in the following update.

**Comment by VenoushCZ** [comment]
Is there only a certain type of triggers hoseSystem operates with? I tried to implement it on the mCompany Factory object MilkStorage but it does not work. No text prompt for attaching of the hose comes up & no errors and warning in the log. Any idea?

**Comment by stijnwop** [comment]
Those mCompany things are not prepared for HS, it will only work with vanilla stuff.
